src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -108,6 +118,16 @@ protected GenInputsAbstract(
   @Option("File that lists methods under test")
   public static Path methodlist = null;
 
+  /**
+   * Automatically add dependencies to the classes under test. For any classes, methods, and
+   * constructors specified by the user, this option adds their argument classes. (The user
+   * specifies them using the {@code --testjar}, {@code --classlist}, {@code --testclass}, and
+   * {@code --methodlist} options.) Recommended to use with {@code --require-covered-classes}
+   * option.
+   */
+  @Option("Add classes that are method/constructor arguments")
+  public static boolean add_dependencies = false;
```

Comment:
Most command-line arguments that add classes/methods under test start with `--test`.  (And the others arguably should, but that change would be disruptive to users.)  I suggest this one should follow the same convention.